homeassistant/components/blink/camera.py

```diff
@@ -121,3 +157,43 @@ def camera_image(
         except TypeError:
             _LOGGER.debug("No cached image for %s", self._camera.name)
             return None
+
+    async def save_recent_clips(self, file_path) -> None:
+        """Save multiple recent clips to output directory."""
+        if not self.hass.config.is_allowed_path(file_path):
+            raise ServiceValidationError(
+                translation_domain=DOMAIN,
+                translation_key="no_path",
+                translation_placeholders={"target": file_path},
+            )
+
+        try:
+            await self._camera.save_recent_clips(output_dir=file_path)
+        except OSError as err:
+            raise ServiceValidationError(
+                str(err),
+                translation_domain=DOMAIN,
+                translation_key="cant_write",
+            ) from err
+
+    async def save_video(self, filename) -> None:
+        """Handle save video service calls."""
+        if not self.hass.config.is_allowed_path(filename):
+            raise ServiceValidationError(
+                translation_domain=DOMAIN,
+                translation_key="no_path",
+                translation_placeholders={"target": filename},
+            )
+
+        try:
+            await self._camera.video_to_file(filename)
+        except OSError as err:
+            raise ServiceValidationError(
+                str(err),
+                translation_domain=DOMAIN,
+                translation_key="cant_write",
+            ) from err
+
+    async def refresh(self):
+        """Call blink to refresh info."""
+        await self.coordinator.api.refresh(force_cache=True)
```

Comment:
Why do we move the refresh service? It's not camera focused.

Reply:
This was why I stopped - didn't make sense to move these but didn't understand how to reference only 1 coordinator.